### PR TITLE
upstream merge 24/10

### PR DESCRIPTION
**Pull request by deltanedas**
## Notable additions
- **mind roles as entities**, this might cause some sneaky bugs but i think i updated everything
- `ftldisk` command for admin abuse
- zombie blood isnt a massive pita anymore
- truncheon does stamina damage on wide swing
- automatic restart after 48h of uptime to prevent memory leak trolling if theres no updates @MilonPL 
- syringe gun for admin abuse

## Notable exclusions
- commented out intellicard from mg locker until AI is enabled
- ghost locator removed from maints spawn

**Changelog**
:cl:
- add: Merged the past few weeks changes from upstream, report any bugs related to your character menu or objectives!

### CODE REVIEW

**Comment by therealDLondon** [comment]
Ima be real I dont like the ghost detector should probably get rid of that lmao

File: Content.Server/Nyanotrasen/Abilities/Psionics/Abilities/MetapsionicPowerSystem.cs
Comment (MilonPL) [inline Content.Server/Nyanotrasen/Abilities/Psionics/Abilities/MetapsionicPowerSystem.cs:1]: if you're touching this file anyways you can remove IPrototypeManager, StatusEffectsSystem, IGameTiming dependencies and the StatusEffect, Prototypes, Timing and Mind usings

File: Content.Shared/DeltaV/Recruiter/RecruiterPenComponent.cs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 using Content.Shared.FixedPoint;
```
Comment (MilonPL) [inline Content.Shared/DeltaV/Recruiter/RecruiterPenComponent.cs]: unused

File: Content.Shared/DeltaV/Recruiter/RecruiterPenComponent.cs [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+using Content.Shared.NPC.Prototypes;
 using Content.Shared.Whitelist;
 using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization;
```
Comment (MilonPL) [inline Content.Shared/DeltaV/Recruiter/RecruiterPenComponent.cs]: serialization unused

**Review by MilonPL — COMMENTED** [review]
everything else looks good

File: Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml [hunk 3]
```diff
@@ -797,7 +797,7 @@
     - state: default
     - state: idseniorofficer
   - type: IdCard # DeltaV - Change senior job titles
-    jobTitle: Senior Officer
+    jobTitle: job-alt-title-senior-officer = Senior Officer
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml]: nvm fucking trolled

Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml]: rea

**Comment by MilonPL** [comment]
ok stupid centcomm map left, i sent what's causing it in #mapping

**Review by MilonPL — APPROVED** [review]
fuck it we ball

**Pull request merged**